functions.php

```diff
@@ -19,6 +19,8 @@
 add_theme_support( 'automatic-feed-links' );
 // enable alignment block options
 add_theme_support( 'align-wide' );
+// disable permalink guessing
+apply_filters( 'do_redirect_guess_404_permalink', false );
```

Comment:
I think you want https://developer.wordpress.org/reference/functions/__return_false/ but otherwise looks good! I haven't tested it yet though.

Reply:
Yes you're absolutely right, my bad. I'll never get used to `'__return_false'` 🙈

Reply:
same though 😅